### DISCUSSION TITLE
Contribution activity isn't showing up in profile

### DISCUSSION THREAD

**Discussion by PranshuC** [community/community#54681] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

In the month of May, I've made some contributions in my project code. The code changes were pushed correctly, the contribution of such code pushes doesn't reflect on the profile page's green chart. Please guide!

**Comment by iamreiyn** [comment, marked as answer]
Hi @PranshuC 

Please read [this](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile) in order to troubleshoot the contributions graph/count of your profile or to know possible reasons of contributions not being counted, thank you.

**Reply by PranshuC** [reply]
Thanks @montymahato , commit email was different from my primary email in GitHub!

**Reply by iamreiyn** [reply]
You're welcome!
